tests/models/mask2former/test_image_processing_mask2former.py

```diff
@@ -223,9 +234,25 @@ def test_with_size_divisor(self):
                 self.assertTrue((pixel_values.shape[-2] % size_divisor) == 0)
 
     def test_call_with_segmentation_maps(self):
-        def common(is_instance_map=False, segmentation_type=None):
+        def common(
+            is_instance_map=False,
+            segmentation_type=None,
+            numpify=False,
+            num_channels=3,
+            input_data_format=None,
+            do_resize=True,
+        ):
+            self.image_processor_tester.num_channels = num_channels
+            self.image_processor_tester.do_resize = do_resize
+            self.image_processor_tester.image_mean = [0.5] * num_channels
+            self.image_processor_tester.image_std = [0.5] * num_channels
```

Comment:
not sure it's a good way to modify class properties in place, that may lead to some hard-to-debug issues later. You can recreate an instance here with an updated image processor dict.